docs/usage.rst

```diff
@@ -105,9 +106,21 @@ To validate a RAML file with Python:
    >>> validate(RAML_FILE)
    InvalidRootNodeError: RAML File does not define an API title.
 
+.. code-block:: python
+
+   >>> from ramlfications import validate
+   >>> RAML_FILE = "/path/to/invalid/no-base-uri-no-title.raml"
+   >>> validate(RAML_FILE)
+   MultipleValidationErrors:
```

Comment:
So I was thinking/re-reviewing this morning (the review a couple of days ago was while traveling and not fully/mentally "present" :smile:).

Correct me if I'm wrong - but the following would happen:

If there is only *one* error within the RAML file, this patch would still raise `MultipleValidationErrors`.

How would you feel renaming the `MultipleValidationErrors` to `InvalidRAML`?  To avoid the incorrect pluralization (e.g. `you have 1 errors`).